### DISCUSSION TITLE
[Feature] Specify network block confirmation time in genesis file

### DISCUSSION THREAD

**Discussion by tommyldev** [0xPolygon/polygon-edge#282] [Ideas] [answered]
### Description

Similar to how geth/puppeth handles the initialization of the genesis files, users can specify block confirmation time:
![image](https://user-images.githubusercontent.com/25253929/141878055-59f02ee2-ee94-4bc2-91ad-ccde0b74ec47.png)

### Expected behaviour

Polygon SDK with IBFT should have a similar feature, will be important for sovereign chains to specify their block throughput based on whatever use-case they have.

**Comment by zivkovicmilos** [comment]
Hey @tommyliu9,

Thank you for reaching out, and I apologize for the late reply.

Specifying a custom block confirmation time is not something that the Polygon SDK currently offers, but having spoken with the team internally, this seems like an interesting feature that we might want to pursue for some future sprint.

Are you using the SDK for a project? We'd love to hear from you regarding the use-case.

**Comment by zivkovicmilos** [comment, marked as answer]
Hey @tommyliu9,

Just to update you on this, we've added this functionality with one of the latest PRs:
https://github.com/0xPolygon/polygon-edge/pull/384
